src/app/EventManagement.h

```diff
@@ -390,7 +390,9 @@ class EventManagement : public DataModel::EventsGenerator
                              EventNumber & generatedEventNumber) override;
 
 private:
-    constexpr EventManagement() = default;
+    // The EventManagement() constructor was made private in pull/38447 but we want the ability to
+    // instantiate this class in test code. Make it public for now until we find a cleaner solution.
+    // constexpr EventManagement() = default;
```

Comment:
```suggestion
    // Note no private constructor in order to preserve the ability to instantiate this class in test code. 
    // Constructor will stay public for now until we find a cleaner solution.
    // constexpr EventManagement() = default;
```

Reply:
Generally I would remove references to other issues or history.
I would actually move this constructor to "public" and have that section there.